spec/index.html

```diff
@@ -15,6 +15,11 @@
           specStatus:           "unofficial",
           // the specification's short name, as in http://www.w3.org/TR/short-name/
           shortName:            "aom",
+          // the specification's repo
+          github: {
+              branch: "gh-pages"
+          ,   repoURL: "WICG/aom"
```

Comment:
@gsnedders I can't say I've noticed the leading comma style before. Is that a new best practice, or just something you prefer personally?

Reply:
This is simply being consistent with the rest of the file: https://github.com/WICG/aom/blob/b12398d735bd39a2523f7d01687fb0cec9f83643/spec/index.html#L43-L71